src/app/components/core/configuration-manager/add-category-child/add-category-child.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+import { NgForm } from '@angular/forms';
+import { AlertService, ConfigurationService } from '../../../../services';
+
+@Component({
+  selector: 'app-add-category-child',
+  templateUrl: './add-category-child.component.html',
+  styleUrls: ['./add-category-child.component.css']
+})
+export class AddCategoryChildComponent implements OnInit {
+  public categoryData: any;
+
+  @Output() notify: EventEmitter<any> = new EventEmitter<any>();
+
+  constructor(
+    private configService: ConfigurationService,
+    private alertService: AlertService
+  ) {}
+
+  ngOnInit() {
+    this.categoryData = {
+      parentCategory: '',
+      child: ''
```

Comment:
childCategory ? to keep it consistent with `parentCategory` or just parent/ child